### PR TITLE
fix(docs): Fix architecture diagram and questions

### LINKED ISSUE #4440: Cleanup multiple docs issues
- [ ] [No cli crate exists](https://docs.iota.org/about-iota/iota-architecture/)
- [ ] Some questions have incorrect answers or are placed on the wrong page
    - [ ] https://docs.iota.org/developer/getting-started/connect
    - [ ] https://docs.iota.org/developer/getting-started/local-network

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Fixed architecture diagram and questions

## Links to any relevant issues

Fixes #4440.

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by lucas-tortora — APPROVED** [review]

File: docs/content/developer/getting-started/local-network.mdx [hunk 1]
```diff
@@ -273,6 +272,3 @@ pnpm add <PATH_TO_YOUR_REPO>/sdk/typescript
 ```
 
 and the locally compiled version of `@iota/iota-sdk` package will be installed for your application.
-
-
-<Quiz questions={questions} />
```
Comment (thibault-martinez) [inline docs/content/developer/getting-started/local-network.mdx:278]: Will we have a quizz later or the page just doesn't deserves one?

File: docs/site/static/json/developer/getting-started/create-a-package.json [hunk 2]
```diff
@@ -1,29 +1,38 @@
 [
   {
-    "questionText": "What is the purpose of the `key` ability in a struct declaration in Move?",
+    "questionText": "What is the command to create a new IOTA Move package called 'my_first_package'?",
     "answerOptions": [
-      { "answerText": "It allows a struct to be stored and transferred.", "isCorrect": true },
```
Comment (thibault-martinez) [inline docs/site/static/json/developer/getting-started/create-a-package.json:5]: The quizz being replaced was a duplicate? It's already somewhere else? We're not actually removing it?

**Review by thibault-martinez — APPROVED** [review]

File: docs/content/developer/getting-started/local-network.mdx (hunk 1 above)
Comment (Dr-Electron) [reply docs/content/developer/getting-started/local-network.mdx:278]: We can add another quiz if you want 😆

File: docs/site/static/json/developer/getting-started/create-a-package.json (hunk 2 above)
Comment (Dr-Electron) [reply docs/site/static/json/developer/getting-started/create-a-package.json:5]: Yes

File: docs/content/developer/getting-started/local-network.mdx (hunk 1 above)
Comment (Dr-Electron) [reply docs/content/developer/getting-started/local-network.mdx:278]: Added a quiz ;)

**Pull request merged**